lms/startup.py

```diff
@@ -91,6 +91,8 @@ def run():
             settings.FEATURES.get('DISABLE_SOLUTIONS_APPS_SIGNALS', False):
         disable_solutions_apps_signals()
 
+    connect_completion_signal()
```

Comment:
We need to add this since signal in edx/completion will not automatically get connected until Hawthorn which has support for django plugins. This code can be removed when the solutions fork upgrades to hawthorn.